cmd/ceremony/main.go

```diff
@@ -258,6 +259,75 @@ func (orc ocspRespConfig) validate() error {
 	return nil
 }
 
+type crlConfig struct {
+	CeremonyType string `yaml:"ceremony-type"`
+	PKCS11       struct {
```

Comment:
It's surprising to me that this PKCS11 stanza, which seems to be shared verbatim between most of the different ceremonies' input configs, isn't shared in code.

Reply:
Some of the PKCS11 stanzas and subtly different, requiring separate structs to allow for using the strict YAML unmarshalling. I think we'll end up using the same struct for a handful of the types though, particularly ones which sign but don't store anything in the HSM. we should clean those up at some point, but that requires a level of churn I think doesn't particularly make sense to do in this change.

Reply:
Yeah, I certainly wouldn't change in this PR. GitHub doesn't seem to have a "leaving a comment but you don't need to resolve it to merge the PR" button?

Reply:
Filed https://github.com/letsencrypt/boulder/issues/4915 to follow up on this.